### ISSUE TITLE
[BUG]: How to collect user email not documented

### ISSUE THREAD

**Issue by AndrewRayCode** [stripe/react-stripe-js#307]
### What happened?

Neither https://stripe.com/docs/stripe-js/react nor https://github.com/stripe/react-stripe-js mention how to collect email or any additional fields.

### Environment

Chrome

**Comment by AndrewRayCode** [comment]
maybe I don't get how to use the docs yet. There are two documentation pages on react-stripe, neither are complete, and the component i'm looking for is documented in the middle of a 10 step flow in a third documentation page that isn't related to react-stripe. I think this bug still stands, but I have what I need now.

**Comment by NickSov** [comment]
Agreed, your documentation is not as complete as it could be. How do we collect the email address via the payment element? Is this even possible?

**Comment by NickSov** [comment]
Looks like this is what you have to do: 
https://stackoverflow.com/questions/70085765/how-to-require-email-on-stripe-element-js
